src/js/actions/SettingsActions.js

```diff
@@ -122,6 +122,16 @@ export default {
         AppActions.app.restart();
     },
 
+    toggleTray: function() {
```

Comment:
That's a fair concern, it's just the other actions are made that way. I'll open an issue for that.